src/languages/c-like.js

```diff
@@ -200,6 +201,58 @@ export default function(hljs) {
     ]
   };
 
+  var TEMPLATE_USE = {
+    begin: /</, end: />/,
+    keywords: CPP_KEYWORDS,
+    contains: EXPRESSION_CONTAINS.concat([
+      // Match left-shift to prevent it from creating a nested `TEMPLATE_USE`.
+      { begin: /<</ },
+      'self'
```

Comment:
This is going to get tripped up on `<` (less than) inside a template I think because you have no way to know if it's an operator or the start of a new TEMPLATE_USE, which is why your one test is failing I think.

Reply:
Also can `<<` Truly never be a nested template use?

Reply:
Yes: each template declaration starts with `template`, and each template instantiation starts with an identifier, followed by a single `<`.